### ISSUE TITLE
Siyuan doesn't want load any pages

### ISSUE THREAD

**Issue by DrTableBasse** [siyuan-note/siyuan#13360]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Can the issue be reproduced with the default theme (daylight/midnight)?

- [X] I was able to reproduce the issue with the default theme

### Could the issue be due to extensions?

- [X] I've ruled out the possibility that the extension is causing the problem.

### Describe the problem

When I want to load any page, I dont have any results.
I delete all plugin, restart the container and have the same problem..
![image](https://github.com/user-attachments/assets/e68902bb-0d7e-4ae7-87e1-2f57e8f2e87b)

### Expected result

Just want to read article for my team x)

### Screenshot or screen recording presentation

![image](https://github.com/user-attachments/assets/40daa154-b1ce-4d5e-a916-6554d1054af6)

### Version environment

```markdown
- Version: latest
- Operating System: Windows 11
- Browser (if used): Firefox Latest
```

### Log file

I 2024/12/03 14:17:24 runtime.go:87: kernel is booting:
    * ver [3.1.14]
    * arch [amd64]
    * os [alpine]
    * pid [1]
    * runtime mode [prod]
    * working directory [/opt/siyuan]
    * read only [false]
    * container [docker]
    * database [ver=20220501]
    * workspace directory [/siyuan/workspace/]
I 2024/12/03 14:17:24 conf.go:130: loaded conf [/siyuan/workspace/conf/conf.json]
I 2024/12/03 14:17:24 runtime.go:139: use network proxy [system]
I 2024/12/03 14:17:24 serve.go:209: kernel [pid=1] http server [0.0.0.0:6806] is booting
I 2024/12/03 14:17:24 conf.go:851: database size [3.9 MB], tree/block count [20/1565]
I 2024/12/03 14:17:24 working.go:193: kernel booted
I 2024/12/03 14:17:25 box.go:77: auto stat [trees=20, blocks=1565, dataSize=67.75 MB, assetsSize=54.26 MB]
I 2024/12/03 14:18:08 session.go:121: auth success [ip=172.20.0.3:56416]

**Comment by DrTableBasse** [comment]
I just tried right now and nothing change lol x)

**Comment by DrTableBasse** [comment]
Latest logs : 

```
main-1  | Creating group siyuan (1000)
main-1  | Creating user siyuan (PUID: 1000, PGID: 1000)
main-1  | Adjusting ownership of /opt/siyuan, /home/siyuan/, and /siyuan/workspace/
main-1  | Starting Siyuan with UID:1000 and GID:1000 in workspace /siyuan/workspace/
main-1  | I 2024/12/03 14:55:20 working.go:147:
main-1  |       ___                                   ___           ___           ___
main-1  |      /  /\        ___           ___        /__/\         /  /\         /__/\
main-1  |     /  /:/_      /  /\         /__/|       \  \:\       /  /::\        \  \:\
main-1  |    /  /:/ /\    /  /:/        |  |:|        \  \:\     /  /:/\:\        \  \:\
main-1  |   /  /:/ /::\  /__/::\        |  |:|    ___  \  \:\   /  /:/~/::\   _____\__\:\
main-1  |  /__/:/ /:/\:\ \__\/\:\__   __|__|:|   /__/\  \__\:\ /__/:/ /:/\:\ /__/::::::::\
main-1  |  \  \:\/:/~/:/    \  \:\/\ /__/::::\   \  \:\ /  /:/ \  \:\/:/__\/ \  \:\~~\~~\/
main-1  |   \  \::/ /:/      \__\::/    ~\~~\:\   \  \:\  /:/   \  \::/       \  \:\  ~~~
main-1  |    \__\/ /:/       /__/:/       \  \:\   \  \:\/:/     \  \:\        \  \:\
main-1  |      /__/:/        \__\/         \__\/    \  \::/       \  \:\        \  \:\
main-1  |      \__\/                                 \__\/         \__\/         \__\/
main-1  | I 2024/12/03 14:55:20 runtime.go:87: kernel is booting:
main-1  |     * ver [3.1.14]
main-1  |     * arch [amd64]
main-1  |     * os [alpine]
main-1  |     * pid [1]
main-1  |     * runtime mode [prod]
main-1  |     * working directory [/opt/siyuan]
main-1  |     * read only [false]
main-1  |     * container [docker]
main-1  |     * database [ver=20220501]
main-1  |     * workspace directory [/siyuan/workspace/]
main-1  | I 2024/12/03 14:55:20 conf.go:130: loaded conf [/siyuan/workspace/conf/conf.json]
main-1  | I 2024/12/03 14:55:20 runtime.go:139: use network proxy [system]
main-1  | I 2024/12/03 14:55:20 serve.go:209: kernel [pid=1] http server [0.0.0.0:6806] is booting
main-1  | I 2024/12/03 14:55:20 conf.go:851: database size [3.8 MB], tree/block count [20/1459]
main-1  | I 2024/12/03 14:55:20 working.go:193: kernel booted
main-1  | I 2024/12/03 14:55:20 box.go:77: auto stat [trees=20, blocks=1459, dataSize=54.8 MB, assetsSize=54.26 MB]
main-1  | W 2024/12/03 14:55:20 serve.go:589: closed an unauthenticated session [172.20.0.3:53542]
main-1  | 2024/12/03 14:55:20 http: response.Write on hijacked connection from github.com/gin-gonic/gin.(*responseWriter).Write (response_writer.go:83)
main-1  | W 2024/12/03 14:55:22 serve.go:589: closed an unauthenticated session [172.20.0.3:53554]
main-1  | 2024/12/03 14:55:22 http: response.Write on hijacked connection from github.com/gin-gonic/gin.(*responseWriter).Write (response_writer.go:83)
main-1  | W 2024/12/03 14:55:23 serve.go:589: closed an unauthenticated session [172.20.0.3:53560]
main-1  | 2024/12/03 14:55:23 http: response.Write on hijacked connection from github.com/gin-gonic/gin.(*responseWriter).Write (response_writer.go:83)
main-1  | W 2024/12/03 14:55:23 serve.go:589: closed an unauthenticated session [172.20.0.3:53572]
main-1  | 2024/12/03 14:55:23 http: response.Write on hijacked connection from github.com/gin-gonic/gin.(*responseWriter).Write (response_writer.go:83)
main-1  | I 2024/12/03 14:55:46 session.go:121: auth success [ip=172.20.0.3:37834]
main-1  | 2024/12/03 14:55:46 http: response.Write on hijacked connection from github.com/gin-gonic/gin.(*responseWriter).Write (response_writer.go:83)
main-1  | 2024/12/03 14:56:02 http: response.Write on hijacked connection from github.com/gin-gonic/gin.(*responseWriter).Write (response_writer.go:83)
main-1  | 2024/12/03 14:56:02 http: response.Write on hijacked connection from github.com/gin-gonic/gin.(*responseWriter).Write (response_writer.go:83)
```

**Comment by DrTableBasse** [comment]
To give more informations, I have authentik in front to create authentification for my users and have Nginx Proxy manager in front too

NPM ---> Authentik ---> Siyuan

So basically i never had problem with siyuan, but i think with the latest, it should create some problems

**Comment by DrTableBasse** [comment]
OK so I check with older version it works !
So If u want, i'll keep this issue open and check in parallele with newest version

**Comment by 88250** [comment]
This is probably not SiYuan's issue. I closed it. Thanks for the feedback.

**Comment by 88250** [comment]
Please remove the middleware from the test to avoid the impact of the environment configuration.

**Comment by DrTableBasse** [comment]
I already do that lol and have the same problem 

I can screenshot again if you want 😄

**Comment by 88250** [comment]
Do you mean there will be authentication issues if you access the IP:port directly?

**Comment by DrTableBasse** [comment]
the logs are the same as i send before... :-/

**Comment by DrTableBasse** [comment]
Yeah i check it and it's OK
For me, it's the new version which as problem as i said before x)

**Comment by 88250** [comment]
We have not received similar feedback from other users so far. We recommend that you check your configuration and environment.

**Comment by TCOTC** [comment]
@DrTableBasse Are you experiencing this issue with Firefox? Does the page load properly using the desktop client?

**Comment by 88250** [comment]
SiYuan does not support Firefox at present.

**Comment by DrTableBasse** [comment]
wtf ? so i should use chrome or other browser ? that what you mean ?

**Comment by 88250** [comment]
Yes, SiYuan has never been compatible with Firefox, and it is highly likely that it will not be compatible in the future.

**Comment by DrTableBasse** [comment]
for which reason isn't compatible ? Oo it's very weird. With the version 3.13 it's OK and works

**Comment by 88250** [comment]
Firefox and Chrome are too different, and we don't have the development resources to make them compatible.

**Comment by jarz76** [comment]
Same issue here. I rolled back to the previous version, and it works fine now. 

Also  just realized that siyuan isn’t compatible with Firefox 😂, as the issue only happens with the latest version.

**Comment by 88250** [comment]
Can you take a look at the error reported in the Console of the developer tools?

**Comment by 88250** [comment]
Try the latest dev version (desktop), which should have fixed the Firefox compatibility issue.

**Comment by AvengineeringCreates** [comment]
I experienced this issue where hierarchy can be modified but page content does not show, no read or write, after downloading the most recent version just yesterday. I'm not super experienced with self-host in general I dismissed it as an issue on my end. I also use Firefox.

It gave a 404 error for `purify.min.js.map`, and threw a bunch of other JS errors for missing references later. What I did was downgrade to v3.1.0 to at least view and write my notes. Neither of those warnings or errors now appear. My setup is too touchy for me to conveniently go back to grab a screenshot, so I am sorry for not contributing much to fixing the error on the developer end, but hopefully it helps someone in the future who just wants to use the notebook now. :)
